scripts/ci

```diff
@@ -204,7 +212,7 @@ postgres() {
 case "$1" in
     lint)
         run_black
-        # run_flake8  TODO: Run this after offending files have been fixed. Ensure appropriate Python version is used.
+        #run_flake8  # TODO: Enable this once we're ready to tackle Flake8 revealed errors
```

Comment:
I thought we were going to enable this on python3 branch?

Reply:
Not until we're ready to fix the errors it reveals (non-trivial)